docs/examples/app-the-new-thing/CHANGELOG.md

```diff
@@ -0,0 +1,100 @@
+# Change Log
+
```

Comment:
Why is this file here? `app-the-new-thing` hasn't been changed yet so this change log is misleading and unnecessary.